.github/workflows/plugins.yml

```diff
@@ -20,6 +20,30 @@ env:
 
 
 jobs:
+  aerospike:
+    runs-on: ubuntu-latest
+    services:
+      aerospike:
+        image: aerospike:ce-6.4.0.3
+        ports: 
+          - "127.0.0.1:3000-3002:3000-3002"
+    env:
+      PLUGINS: aerospike
+      SERVICES: aerospike
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/testagent/start
+      - uses: ./.github/actions/node/setup
+      - run: yarn install --ignore-engines
+      - uses: ./.github/actions/node/14
```

Comment:
Why not just use `oldest` and `latest` like other plugins?

Reply:
latest will test > node 20, will change it to oldest once the pr's approved, it's mainly for my own testing since older aerospike versions don't work on arm